#request constants

#request fields
int32 pos
Neopixel neopixel
---
#response constants

#response fields
bool state
